src/stories/Tutorials/BaseTutorials/index.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import ProviderWrapper from "../../../helpers/providerWrapper";
+import BaseTutorial from "../../../components/Tutorials/MyTutorials/BaseTutorialsComponent/index";
+
+export default {
+  title: "Tutorials/BaseTutorial",
+  component: BaseTutorial,
+};
+
+const Template = (args) => (
+  <ProviderWrapper>
+    <MemoryRouter>
+      <BaseTutorial {...args} />
+    </MemoryRouter>
+  </ProviderWrapper>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  owner: "rijusougata31",
+  ownerName: "riju",
+  users: [
+    {
+      featured_image: "",
+      icon: "",
+      owner: "rijusougata31",
+      summary: "a",
+      title: "CodeLabz - FrontEnd Improvements",
```

Comment:
remove personal information and put some dummy data. Better to have something that represents SCoRe Lab here.

Reply:
This are the usernames from firebase. I am creating a user to represent scorelab.I will replace this with scorelab username.

Reply:
You do not need to create a new user in firebase particularly for this purpose. What I suggest was use dummy data that represent SCoRe Lab. And since you are passing the data as props, the component should rendered correctly.

Reply:
yeah got your point .. replaced my data with dummy data